src/Menu/Menu.jsx

```diff
@@ -174,12 +175,14 @@ class Menu extends React.Component {
   open() {
     if (this.props.onOpen) this.props.onOpen();
     this.setState({ expanded: true });
+    document.addEventListener('touchend', this.onDocumentClick, true);
```

Comment:
Could you add a comment explaining that touchend is for mobile?

Reply:
Yes will do